apps/linode-marketplace-influxdb/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# set_hostname
+- name: setting up hostname
+  import_role:
+    name: hostname
+
+# write _domain to vars.
+- name: write _domain to vars
+  lineinfile: 
+    insertafter: EOF
+    path: group_vars/linode/vars
+    line: | 
+      _domain: {{ _domain }}
+  when: _domain is defined 
+
+# dns creation
+- name: create dns A record
+  import_role:
+    name: create_dns_record
+  when: 
+    - token_password is defined
+    - default_dns is not defined
+
+# sets customer supplied ssh pubkey to limited user
+- name: set ssh pubkey
+  import_role:
+      name: sshkey
+  when: 
```

Comment:
The when cause can be removed, we've moved to just copying the .ssh/authorized_keys file from root to user if they use the cloud manager ssh key feature

Reply:
removed in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)